client/landing/stepper/README.md

```diff
@@ -38,6 +38,8 @@ To create a flow, you only have to implement `useSteps` and `useStepNavigation`.
 
 There is also an optional `useSideEffect` hook. You can implement this hook to run any side-effects to the flow. You can prefetch information, send track events when something changes, etc...
 
+There is an optional `isSignupFlow` flag that _MUST be implemented for signup flows_ (generally where a new site may be created), as well as an optional `useSignupStartEventProps()` hook. The `isSignupFlow` flag controls whether we'll trigger a `calypso_signup_start` Tracks event when the flow starts, while the `useSignupStartEventProps()` hook makes it possible to add custom field to the event that will get logged.
```

Comment:
With the changes to make `isSignupFlow` required, the wording here should be updated to match.
```suggestion
There is a required `isSignupFlow` flag that _MUST be set to `true` for signup flows_ (generally where a new site may be created), and should be `false` for other flows. There is also an optional `useSignupStartEventProps()` hook. The `isSignupFlow` flag controls whether we'll trigger a `calypso_signup_start` Tracks event when the flow starts, while the `useSignupStartEventProps()` hook makes it possible to add custom fields to the event that will get logged.
```

Reply:
Updated in edb04500ed543fb96d408c68cfdef6e77631a7c6.